src/ecalc/libraries/libecalc/common/libecalc/core/graph_result.py

```diff
@@ -343,3 +388,43 @@ def get_results(self) -> EnergyCalculatorResult:
             emission_results=self.emission_results,
             variables_map=self.variables_map,
         )
+
+    def get_pressures_from_temporal_models(
+        self, energy_usage_model: dict, regularity: dict, pressure_type: CompressorPressureState
+    ) -> Dict[datetime.datetime, TimeSeriesFloat]:
+        """Extract compressor input pressures from temporal models.
+
+        :param energy_usage_model: dictionary of temporal energy models
+        :param regularity: regularity for the actual component
+        :param input_pressure_type: type of pressure to evaluate (inlet- or outlet pressure)
+        :return: inlet- and outlet input pressure time series
+        """
+
+        pressures = {}
+
+        for time_value, model in energy_usage_model.items():
+            pressure = model.suction_pressure
+            if input_pressure_type.value == input_pressure_type.outlet_pressure:
```

Comment:
```suggestion
            if input_pressure_type.value == CompressorPressureState.OUTLET_PRESSURE:
```
When checking against enum, call Class directly. Also, forgot to mention that enums should be uppercase because they are constants)